datafusion-examples/examples/csv_sql_streaming.rs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can we also please add a line in https://github.com/apache/arrow-datafusion/tree/main/datafusion-examples#single-process about this example?

Reply:
Sure, added